packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -302,15 +302,18 @@ class FileCollection extends DocumentCollection {
    */
   async createFile(
     data,
-    { name, dirId = '', executable, metadata, ...options } = {}
+    { name, dirId, executable, metadata, ...options } = {}
```

Comment:
I agree that this is confusing that sometime we pass a dirId and sometime not. But I'm not confortable with this kind of BC without previous notice. We should warn the dev, and set the default value to `''` if needed. And in a near future remove this warning and really break the stuff.

Reply:
Yes, I should have checked the doc, I was not sure that without folderId this would work but it works.

https://docs.cozy.io/en/cozy-stack/files/#post-filesdir-id

>  When it’s not given, the directory is created at the root of the virtual file system.

I removed the commit.